src/plugins/slack/models.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+/**
+ * Data model for fetched data from Slack API
+ * Ways of interaction include 
+ * - Channels: public, private, group DMs and DMs
+ * - Messages: Conversations inside channels
+ * - Threads: replies to a message inside a channel
+ * - Reacts: reactions left by other members of a channel on a message
+ */
+
+export type SlackToken = string;
```

Comment:
might be good to add a docstring here clarifying the purpose of the SlackToken type, and link to where tokens can be generated